src/Magnum/EigenIntegration/Test/IntegrationTest.cpp

```diff
@@ -215,6 +231,78 @@ void IntegrationTest::matrixMatrix() {
     CORRADE_COMPARE_AS((cast<Eigen::Matrix<double, 2, 3>>(a)), b, EigenType);
 }
 
+
+void IntegrationTest::stridedArrayView() {
+    float data[4*5];
+    for(int i = 0; i < 4*5; ++i) data[i] = float(i);
+    Containers::StridedArrayView2D<float> view1(data, {4, 5});
+    auto mapped = arrayCast(view1);
+    auto view2 = arrayCast(mapped);
+    for(int i = 0; i < 4; ++i) {
+        for(int j = 0; j < 5; ++j) {
+            CORRADE_COMPARE(view1[i][j], view2[i][j]);
+        }
+    }
+}
```

Comment:
This could use an additional test case that 

1. uses a `flipped()`, `broadcasted()` and `transposed()` array on input to test all cases of a positive, zero and negative stride
2. converts to Eigen, verifies a few random picks (`a[2][3] == b[2][3]`)
3. converts back

and then verifies that `data()`, `size()` and `stride()` are the same as on input. In particular I'm not sure if Eigen even supports zero strides (I hope it supports negative strides at least).

Reply:
So I researched this a bit, and it is possible to incorporate negative strides by mapping those to Eigen::Reverse expressions I think.
Unfortunately I dont think it is possible to cast broadcasted ArrayViews, since Eigen handels braodcasting differently. E.g. to add a vector to a matrix one turns the matrix into a special kind of broadcast expression but the vector is not changed in any way 
```
Eigen::MatrixXf m; 
Eigen::VectorXf v;
m.rowwise() += v;
```
So if you agree I would just assert on zero strides.

Reply:
Ok actually the Reverse expression are not well suited since everything needs to be known at compile time. I found that negative strides are actually supported out of the box since relatively recently : https://eigen.tuxfamily.org/bz/show_bug.cgi?id=747 .

Reply:
Looking at the commit log, this seems to be done and fully supported, right? :)